man/ronin-archive.1.md

```diff
@@ -0,0 +1,46 @@
+# ronin-archive 1 "2024-01-18" Ronin "User Manuals"
+
+## NAME
+
+ronin-archive - Archive the files.
+
+## SYNOPSIS
+
+`ronin archive` [*options*] [`-t` \| `--tar`] [`-z` \| `--zip`]  {`-o`,`--output` *FILENAME* } [*FILE* ...]
+
+## DESCRIPTION
+
+Archive the files.
+
+## ARGUMENTS
+
+[*FILE* ...]
```

Comment:
Remove the `[` and `...]`. Argument definitions will refer to a single argument value.